HackingWithSwift/SwiftUI/Project1/WeSplit/WeSplit/ContentView.swift

```diff
@@ -0,0 +1,85 @@
+//
```

Comment:
Let’s do a challenge: provide explicit types for all your constants, variables, etc. This will help you understand your code better and also reinforce good practices.